app/scripts/modules/titus/src/serverGroup/configure/wizard/pages/disruptionBudget/WindowPicker.tsx

```diff
@@ -0,0 +1,200 @@
+                >
+                  <div className="sp-padding-m-right" style={{ color: 'var(--color-text-on-dark)', fontSize: '12px' }}>
+                    <div>
+                      <b>{timeWindow.days.map(d => d.substr(0, 3)).join(', ')}</b>
+                    </div>
+                    <div>
+                      <em>
+                        {w.startHour}:00 - {w.endHour}:00 {timeWindow.timeZone}
+                      </em>
+                    </div>
+                  </div>
+                  {!isDisabled && (
+                    <a
+                      className="clickable"
+                      style={{ color: 'var(--color-text-on-dark)' }}
+                      onClick={() => this.removeWindow(i, i2)}
+                    >
+                      <span className="glyphicon glyphicon-remove" />
+                    </a>
+                  )}
+                </div>
+              </div>
+            ));
+          })}
+        </div>
+      </div>
+    );
+  };
+
+  private NewWindowComponent = () => {
+    if (!this.state.addingNewWindow) {
+      return (
+        <div className="sp-formItem">
+          <div className="sp-formItem__left" />
+          <div className="sp-formItem__right sp-margin-l-bottom">
+            <a className="button primary" onClick={this.configureNewWindow}>
+              <i className="fas fa-plus-circle" /> Define New Window
+            </a>
+          </div>
+        </div>
+      );
+    }
+    return (
+      <Formik<IJobTimeWindowForm>
+        initialValues={{
+          days: [],
+          startHour: 10,
+          endHour: 16,
+          timeZone: get(this.props.formik.values.disruptionBudget, 'timeWindows[0].timeZone', 'PST'),
+        }}
+        onSubmit={this.addNewWindow}
+        render={formik => (
+          <>
+            <FormikFormField
+              name="days"
+              label="Define a New Window"
+              input={props => <ChecklistInput {...props} options={DAYS} />}
+            />
+            <FormikFormField
+              name="startHour"
+              label="Start"
+              fastField={false}
+              input={props => (
+                <div>
+                  <NumberInput {...props} min={0} max={formik.values.endHour} />
+                  <HelpField expand={true} content="hour (0-24)" />
+                </div>
+              )}
+            />
+            <FormikFormField
+              name="endHour"
+              label="End"
+              fastField={false}
+              input={props => (
+                <div>
+                  <NumberInput {...props} min={formik.values.startHour} max={23} />
+                  <HelpField expand={true} content="hour (0-24)" />
+                </div>
+              )}
+            />
+            <FormikFormField
+              name="timeZone"
+              label="Timezone"
+              input={props => <ReactSelectInput {...props} clearable={false} stringOptions={['PST', 'UTC']} />}
```

Comment:
We're starting with these two time zones, and will add more if folks request them.